src/main.rs

```diff
@@ -1261,6 +1262,15 @@ async fn process_account_list(
                         .or_default()[lot.acquisition.when.month0() as usize / 3]
                         .income += lot.income(account.token);
 
+                    if let LotAcquistionKind::EpochReward { epoch: _, slot: _ } =
```

Comment:
Ah, I suspect you may be holding `sys` differently than I am.  Whenever I have a different `LotAcquistionKind` I always provide the post-tax acquisition price to `sys`, so effectively all future income is rewards income.  But perhaps there's a different use case that I haven't considered. DM me up on Slack if you're rather not discuss the particulars in public though :)